### ISSUE TITLE
Some images are too big for serving on web

### ISSUE THREAD

**Issue by xristy** [BuddhistDigitalResourceCenter/library-issues#29]
There are images that either need to be in a .../web/... dir or otherwise compressed. For example the first image in W1KG14667.

There needs to be an audit script that processes the entire archive producing a report for images larger than some threshold and needing a .../web/... dir. The threshold could be solved perhaps by reporting images that are 500KB <= 1MB, 1MB <= 2MB, and > 2MB or similar.

In other words create some bins and put images in them. Maybe even 500 KB is too large for a minimum. Maybe it should be 200KB
